src/SFA.DAS.Commitments.Support.SubSite/DependencyResolution/DefaultRegistry.cs

```diff
@@ -0,0 +1,124 @@
+// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="DefaultRegistry.cs" company="Web Advanced">
+// Copyright 2012 Web Advanced (www.webadvanced.com)
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace SFA.DAS.Commitments.Support.SubSite.DependencyResolution
+{
+    using FluentValidation;
+    using MediatR;
+    using Microsoft.Azure;
+    using SFA.DAS.Commitments.Application.Queries.GetApprenticeshipsByUln;
+    using SFA.DAS.Commitments.Application.Queries.GetCommitment;
+    using SFA.DAS.Commitments.Domain.Data;
+    using SFA.DAS.Commitments.Domain.Interfaces;
+    using SFA.DAS.Commitments.Infrastructure.Data;
+    using SFA.DAS.Commitments.Infrastructure.Data.Transactions;
+    using SFA.DAS.Commitments.Infrastructure.Logging;
+    using SFA.DAS.Commitments.Infrastructure.Services;
+    using SFA.DAS.Commitments.Support.SubSite.Configuration;
+    using SFA.DAS.Commitments.Support.SubSite.GlobalConstants;
+    using SFA.DAS.Commitments.Support.SubSite.Mappers;
+    using SFA.DAS.Commitments.Support.SubSite.Models;
+    using SFA.DAS.Commitments.Support.SubSite.Orchestrators;
+    using SFA.DAS.Commitments.Support.SubSite.Validation;
+    using SFA.DAS.Configuration;
+    using SFA.DAS.Configuration.AzureTableStorage;
+    using SFA.DAS.HashingService;
+    using SFA.DAS.Learners.Validators;
+    using SFA.DAS.NLog.Logger;
+    using StructureMap;
+    using StructureMap.Graph;
+    using System.Web;
+
+    public class DefaultRegistry : Registry
+    {
+      
+        private const string Version = "1.0";
+
+        public DefaultRegistry()
+        {
+            Scan(
+                 scan =>
+                 {
+                     scan.TheCallingAssembly();
+                     scan.WithDefaultConventions();
+
+                     scan.AssembliesFromApplicationBaseDirectory(a => a.GetName().Name.StartsWith("SFA.DAS"));
+                     scan.RegisterConcreteTypesAgainstTheFirstInterface();
+                     scan.ConnectImplementationsToTypesClosing(typeof(AbstractValidator<>));
```

Comment:
The validators should be set to have a Singleton lifetime, otherwise they (in this case ApprenticeshipsSearchQueryValidator) will run the rule setup continually and unnecessarily.